docs/pages/components/token.md

```diff
@@ -280,3 +285,75 @@ To show focus on whole element, it's needed to add `.is-focus` into container.
 </div>
 {% endcapture %}
 {% include display-component.html component=compact-tokenizer %}
+
+## Tokenizer - Scrollable
+The Compact tokenizer is for large screens.
```

Comment:
This is the cozy tokenizer.  And the user won't have tab keys in cozy mode (phone/tablet).  In compact (desktop) mode they will have the "____more" text which when clicked opens the popover list containing the hidden tokens.

Reply:
In my my approach, there is added `--scrollable`, which can be added in cozy and compact mode.